libraries/standard/mqtt/cbmc/proofs/MQTT_GetPacketId/MQTT_GetPacketId_harness.c

```diff
@@ -0,0 +1,17 @@
+/*
+ * Insert copyright notice
+ */
+
+/**
+ * @file MQTT_GetPacketId_harness.c
+ * @brief Implements the proof harness for MQTT_GetPacketId function.
+ */
+
+#include "mqtt.h"
+
+void harness()
+{
+    MQTTContext_t context;
```

Comment:
Let's investigate the malloc flag for cbmc and consider mallocing a possibly null pointer to a MQTTContext_t.